### ISSUE TITLE
Configurable name of organization shown on web interface

### ISSUE THREAD

**Issue by Volker-K** [iris-connect/iris-client#160]
We will host the IRIS client for three health departments. 
Therefore, we would like to "personalise" those three instances by displaying the name of the health department in the header or elsewhere, so that users can notice themselves if they want to log in to the wrong instance.
This requirement will end up being important for all users, as presumably all health departments will also be running a staging environment and users need to be prevented from accidentally logging into the staging environment for productive work. 
For IT service providers with multiple health departments among their clients, this also applies - as it does for us - to the client names.

**Comment by hey-johnnypark** [comment]
Hi @Volker-K , we'll discuss it in the team and come back to you.

**Comment by hey-johnnypark** [comment]
@Volker-K The task has been approved by our team. You will be notified once it has been released.

**Comment by jekutzsche** [comment]
@lucky-lusa 
Do you know of a configuration (e.g., an environment variable) from which we can read the name of the health department? I don't see anything so far. However, the following ticket fits here: https://github.com/iris-connect/eps/issues/20.

**Comment by lucky-lusa** [comment]
Yes, when iris-connect/eps#20 is done we can display the real health departments name.
